packages/cozy-harvest-lib/src/cozy-ui.d.ts

```diff
@@ -11,4 +11,19 @@ declare module 'cozy-ui/transpiled/react/*' {
   export const useI18n: () => {
```

Comment:
I really don't like this whole file, maybe it's time to add it directly in cozy-ui repo